extensions/amp-access-laterpay/0.1/laterpay-impl.js

```diff
@@ -492,11 +492,11 @@ export class LaterpayVendor {
    */
   formatPrice_(priceValue) {
     const value = priceValue / 100;
-    const props = {
+    const opts = {
```

Comment:
This change suggests that we don't want other files to use `props` outside this intended use case -- if this is the case, what do you think about adding a lint rule about naming a variable `'prop'` to make this more clear? Or as @choumx  suggested elsewhere constraining the set of files that this is applied to.